cylc/flow/config_schema.py

```diff
@@ -0,0 +1,660 @@
+            'global init-script': [VDR.V_STRING, ''],
+            'copyable environment variables': [VDR.V_STRING, ''],
+            'retrieve job logs': [VDR.V_STRING, ''],
+            'retrieve job logs command': [VDR.V_STRING, ''],
+            'retrieve job logs max size': [VDR.V_STRING, ''],
+            'retrieve job logs retry delays': [VDR.V_STRING, ''],
+            'task event handler retry delays': [VDR.V_STRING, ''],
+            'tail command template': [VDR.V_STRING, ''],
+            'batch systems': {
+                '__MANY__': {
+                    'err tailer': [VDR.V_STRING, ''],
+                    'out tailer': [VDR.V_STRING, ''],
+                    'err viewer': [VDR.V_STRING, ''],
+                    'out viewer': [VDR.V_STRING, ''],
+                    'job name length maximum': [VDR.V_STRING, ''],
+                    'execution time limit polling intervals': [
+                        VDR.V_STRING, ''
+                    ],
+                }
+            }
+        },
+    },
+    'monitor': {
+        'sort order': [VDR.V_STRING, 'definition', 'alphanumeric'],
+    },
+    'scheduling': {
+        'initial cycle point': [VDR.V_CYCLE_POINT],
+        'final cycle point': [VDR.V_STRING],
+        'initial cycle point constraints': [VDR.V_STRING_LIST],
+        'final cycle point constraints': [VDR.V_STRING_LIST],
+        'hold after point': [VDR.V_CYCLE_POINT],
+        'cycling mode': (
+            [VDR.V_STRING, Calendar.MODE_GREGORIAN] +
+            list(Calendar.MODES) + ["integer"]),
+        'runahead limit': [VDR.V_STRING],
+        'max active cycle points': [VDR.V_INTEGER, 3],
+        'spawn to max active cycle points': [VDR.V_BOOLEAN],
+        'queues': {
+            'default': {
+                'limit': [VDR.V_INTEGER, 0],
+                'members': [VDR.V_STRING_LIST],
+            },
+            '__MANY__': {
+                'limit': [VDR.V_INTEGER, 0],
+                'members': [VDR.V_STRING_LIST],
+            },
+        },
+        'special tasks': {
+            'clock-trigger': [VDR.V_STRING_LIST],
+            'external-trigger': [VDR.V_STRING_LIST],
+            'clock-expire': [VDR.V_STRING_LIST],
+            'sequential': [VDR.V_STRING_LIST],
+            'exclude at start-up': [VDR.V_STRING_LIST],
+            'include at start-up': [VDR.V_STRING_LIST],
+        },
+        'xtriggers': {
+            '__MANY__': [VDR.V_XTRIGGER],
+        },
+        'graph': {
+            '__MANY__': [VDR.V_STRING],
+        },
+    },
+    'runtime': {
+        '__MANY__': {
+            'inherit': [VDR.V_STRING_LIST],
+            'init-script': [VDR.V_STRING],
+            'env-script': [VDR.V_STRING],
+            'err-script': [VDR.V_STRING],
+            'exit-script': [VDR.V_STRING],
+            'pre-script': [VDR.V_STRING],
+            'script': [VDR.V_STRING],
+            'post-script': [VDR.V_STRING],
+            'extra log files': [VDR.V_STRING_LIST],
```

Comment:
Not sure if this is still used or not.

Reply:
I'll check...

Reply:
It is harmless. Let's leave it for now.

Reply:
This is one is useful for exposing additional log files for easy viewing in gcylc. I remember reporting it broken and fixed by @oliver-sanders in #2515.

Reply:
(I have occasionally used this myself)